app/helpers/manage_consents_helper.rb

```diff
@@ -10,4 +10,21 @@ def form_method_for(consent)
   def include_clone_fields_for(consent)
     consent.recorded?
   end
+
+  # rubocop:disable Rails/HelperInstanceVariable
```

Comment:
I didn't feel like passing in 5 different arguments to `back_link_path`.

Rubocop's docs say that [Rails/HelperInstanceVariable](https://www.rubydoc.info/gems/rubocop-rails/RuboCop/Cop/Rails/HelperInstanceVariable) is useful to make helpers more reusable, but this helper is veeery specific to this controller. So I decided to overrule it.